### PR TITLE
Home Work 6

### PR DESCRIPTION
**Pull request by TanyaB777**

### CODE REVIEW

File: src/main/java/hw_6/Task1.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+        System.out.println("Non Static");
+        System.out.printf("x = %d y = %d \n", x, y);
+
+        x = 10;
+        y = 15;
+
+        System.out.printf("x = %d y = %d", x, y);
+    }
+
+    private static void StaticMethod() {
```
Comment (AntBelyi) [inline src/main/java/hw_6/Task1.java]: method name should begin from the small letter

File: src/main/java/hw_6/Task1.java [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+    }
+
+    int x = 3;
+    int y = 4;
+
+    static int x1 = 5;
+    static int y1 = 8;
+
+    private void NonStaticMethod() {
```
Comment (AntBelyi) [inline src/main/java/hw_6/Task1.java]: method name should begin from the small letter

File: src/main/java/hw_6/Task1.java [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+        Task1 task = new Task1();
+        task.NonStaticMethod();
+
+        StaticMethod();
+
+    }
+
+    int x = 3;
+    int y = 4;
```
Comment (AntBelyi) [inline src/main/java/hw_6/Task1.java]: Where is access modifiers?) Please add in all places

File: src/main/java/hw_6/Task1.java [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+public class Task1 {
+    public static void main(String[] args) {
+
+        Task1 task = new Task1();
+        task.NonStaticMethod();
+
+        StaticMethod();
+
+    }
```
Comment (AntBelyi) [inline src/main/java/hw_6/Task1.java]: Task1 Should be a separate class. So, please create a separate App class for main method and move this main method there

File: src/main/java/hw_6/Task1.java (hunk 3 above)
Comment (TanyaB777) [reply src/main/java/hw_6/Task1.java]: fixed

File: src/main/java/hw_6/Task1.java (hunk 2 above)
Comment (TanyaB777) [reply src/main/java/hw_6/Task1.java]: fixed

File: src/main/java/hw_6/Task1.java (hunk 1 above)
Comment (TanyaB777) [reply src/main/java/hw_6/Task1.java]: fixed

File: src/main/java/hw_6/Task1.java (hunk 4 above)
Comment (TanyaB777) [reply src/main/java/hw_6/Task1.java]: fixed

**Pull request merged**